regcore/tests/db_django_models_tests.py

```diff
@@ -75,33 +75,33 @@ def setUp(self):
         self.dml = DMLayers()
 
     def test_get_404(self):
-        self.assertIsNone(self.dml.get('namnam', 'verver:lablab'))
+        self.assertIsNone(self.dml.get('namnam', 'cfr', 'verver:lablab'))
```

Comment:
You're right, though it doesn't affect this particular test. I'll add some docs, too.